tests/Unit/Infrastructure/FileSyncer/PhpFileSyncerTest.php

```diff
@@ -85,44 +101,43 @@ public function testCopy($source, $givenDestination, $expectedDestination, $give
         self::assertSame((string) $expectedTimeout, ini_get('max_execution_time'), 'Correctly set process timeout.');
     }
 
-    public function providerCopyWithOptionalParams(): array
+    public function providerSyncWithOptionalParams(): array
     {
         return [
             [
                 'source' => 'lorem',
                 'givenDestination' => '',
                 'expectedDestination' => '',
                 'givenExclusions' => null,
-                'expectedExclusions' => [''],
+                'expectedExclusions' => [],
```

Comment:
Subtle point, but why did this change?

Reply:
Because the destination is no longer added to `$exclusions` but passed separately to `Finder::notPath()`, which makes the production code a little more readable. See https://github.com/php-tuf/composer-stager/pull/15/files#diff-6c0b3424be08df4d637f676cbbeaf7dcbff55a8b0258de16b408385b4b74c4bfR103-R106.